### PR TITLE
fix: fix infinite spinner in Enum component when target dataset doesn…

### PR DESCRIPTION
**Pull request by esasova**
…'t exist

Before creating a PR, please check that:
- [ ] Code is clean
- [ ] Tests are still working (cypress tests and `yarn test`)
- [ ] Changes don't cause new errors or warnings in browser console
- [ ] Documentation is up-to-date
- [ ] Breaking changes are clearly identified with [conventional commits](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)

### CODE REVIEW

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

File: src/views/Scenario/Scenario.js [hunk 1]
```diff
@@ -242,7 +242,7 @@ const Scenario = () => {
   const validationAreaDivider =
     userPermissionsOnCurrentScenario.includes(ACL_PERMISSIONS.SCENARIO.VALIDATE) ||
     currentScenarioData?.validationStatus !== SCENARIO_VALIDATION_STATUS.DRAFT ? (
-      <Divider orientation="vertical" flexItem />
+      <Divider orientation="vertical" flexItem sx={{ marginX: 2 }} />
```
Comment (nborde-CSM) [inline src/views/Scenario/Scenario.js:245]: You can use `mx` who is little shorter but it is at you convenience...

**Review by nborde-CSM — COMMENTED** [review]
One remark but LGTM :+1:

File: src/views/Scenario/Scenario.js (hunk 1 above)
Comment (esasova) [reply src/views/Scenario/Scenario.js:245]: good idea, thanks ! :pray:

**Pull request merged**